activerecord/lib/active_record/core.rb

```diff
@@ -699,6 +700,11 @@ def strict_loading!(value = true, mode: :all)
 
     attr_reader :strict_loading_mode
 
+    # Returns +true+ if the record uses strict_loading with +:all+ mode enabled.
+    def strict_loading_all?
+      @strict_loading_mode == :all
```

Comment:
Should not this only return true if `strict_loading` is true?

Reply:
I can change both if you'd like.

Reply:
👋 gentle bump in case you want the method body changed here!